recording/lib/storage/file.ex

```diff
@@ -0,0 +1,24 @@
+defmodule Membrane.RTC.Engine.Endpoint.Recording.Storage.File do
+  @moduledoc """
+  `Membrane.RTC.Engine.Endpoint.Recording.Storage` implementation that saves the stream to files locally.
+  """
+  @behaviour Membrane.RTC.Engine.Endpoint.Recording.Storage
+
+  @impl true
+  def get_sink(opts) do
+    location = Path.join(opts.output_dir, opts.filename)
+    File.mkdir_p!(opts.output_dir)
+    File.touch!(location)
```

Comment:
JFYI, File.Sink creates a file if it doesn't exist, but only the file, not the whole path